porch/pkg/apiserver/apiserver.go

```diff
@@ -76,6 +76,7 @@ type ExtraConfig struct {
 	CacheDirectory        string
 	FunctionRunnerAddress string
 	DefaultImagePrefix    string
+	RepoSyncFrequency     int64
```

Comment:
We should change this to a time.Duration

Reply:
Done